### ISSUE TITLE
Forgot to call reset_parameters() in __init__

### ISSUE THREAD

**Issue by arkravchuk** [amirhfarzaneh/lsoftmax-pytorch#7]
Hi, @amirhfarzaneh !

In ```__init__```, you forgot to call method ```reset_parameters```, because of this weight of the linear layer -- it's just zeros and the  loss goes to nan

**Comment by WeihanLikk** [comment]
Maybe you should call this method by yourself. You can look for his examples

**Comment by brianw0924** [comment]
@akravchuk97 

I got the same problem, the loss is always nan

where should i add the reset_parameters??

**Comment by arkravchuk** [comment]
@brianw0924 

You can call `reset_parameters` after  loss initialization.

```python
loss = LSoftmaxLinear(...)
loss.reset_parameters()
```

**Comment by brianw0924** [comment]
@akravchuk97 

i think we implement in different way...

actually, i don't really understand how to use this loss function,

I still use CrossEntropyLoss() as my criterion, and I put reset_parameters in the __init__ of LSoftmaxLinear

and I replace my output layer as LSoftmaxLinear(last layer dim, output dim)

is it right?

i feel that the accuracy is not better (i'm training phoneme classification)
